package.json

```diff
@@ -40,6 +41,7 @@
     "eslint-plugin-jest": "^21.27.2",
     "eslint-plugin-node": "^4.2.3",
     "eslint-plugin-promise": "^3.8.0",
+    "eslint-plugin-react": "^7.12.4",
     "eslint-plugin-standard": "^3.1.0",
```

Comment:
do we need `eslint-plugin-react` for something?

Reply:
See https://github.com/transloadit/uppy/commit/165ca034164deb2863d659aa0d6de5d7bba97cbe for rationale

Reply:
all right, much thanks for the explanation!